Empty docs in Xcode 11 (despite latest sourcekitten)

Hello,

We are using `sourcekitten 0.29.0`, but we encounter the same issue that the one described in https://github.com/realm/jazzy/issues/1108 … despite that issue supposedly being fixed by https://github.com/jpsim/SourceKitten/pull/618 and the fix being shipped in `0.26.0`

> Side note: we're invoking `sourcekitten` independently (because we want to merge multiple projects docs in a single docset, so we invoke `sourcekitten` ourselves on the various projects, and only then use `jazzy --sourcekitten-sourcefile docs.json …`; which is also why I posted here and not in jazzy repo)

Not sure if we're holding it wrong, or if the issue was just not entirely fixed by https://github.com/jpsim/SourceKitten/pull/618 in the first place at we hoped…
Any idea @jpsim @johnfairh ?

```sh
$ sourcekitten version
0.29.0
$ xcodebuild -version
Xcode 11.1
Build version 11A1027
```

Empty docs with Xcode 11, despite using sourcekitten 0.29 which is supposed to have that fix:
```sh
$ rm -rf /tmp/sdkdoc && mkdir /tmp/sdkdoc && sourcekitten doc --module-name BabylonSDK -- -workspace ../Babylon.xcworkspace -scheme BabylonSDK -derivedDataPath /tmp/sdkdoc
Running xcodebuild
Checking xcodebuild -showBuildSettings
Assuming New Build System is used.
[]
```

Fix to pass `USE_SWIFT_RESPONSE_FILE=NO` suggested in https://github.com/realm/jazzy/issues/1108 works (but feels like a workaround, especially now that this is supposed to be fixed since 0.26):
```sh
$ rm -rf /tmp/sdkdoc && mkdir /tmp/sdkdoc && sourcekitten doc --module-name BabylonSDK -- USE_SWIFT_RESPONSE_FILE=NO -workspace ../Babylon.xcworkspace -scheme BabylonSDK -derivedDataPath /tmp/sdkdoc
Running xcodebuild
Checking xcodebuild -showBuildSettings
Assuming New Build System is used.
Parsing AppointmentBusinessController.swift (1/272)
Parsing AppointmentArchives.swift (2/272)
Parsing AppointmentCancellationReason.swift (3/272)
^C
```
Using Xcode 10.3 (via `DEVELOPER_DIR=…`) works like it used to, even without the `USE_SWIFT_RESPONSE_FILE=NO` workaround. So issue is only present since we migrated to Xcode 11:
```sh
$ rm -rf /tmp/sdkdoc && mkdir /tmp/sdkdoc && DEVELOPER_DIR=/Applications/Xcode-10.3.app sourcekitten doc --module-name BabylonSDK -- -workspace ../Babylon.xcworkspace -scheme BabylonSDK -derivedDataPath /tmp/sdkdoc
Running xcodebuild
Checking xcodebuild -showBuildSettings
Assuming New Build System is used.
Parsing AppointmentBusinessController.swift (1/272)
Parsing AppointmentArchives.swift (2/272)
Parsing AppointmentCancellationReason.swift (3/272)
^C
```

> What was the error from xcodebuild when it failed? 

Here's the output of `xcodebuild -workspace Babylon.xcworkspace -scheme BabylonSDK -derivedDataPath /tmp/sdkdoc`:

<details>
<summary>It starts normally</summary>

```
User defaults from command line:
    IDEDerivedDataPathOverride = /tmp/sdkdoc

note: Using new build system
note: Planning build
note: Constructing build description
CreateBuildDirectory /tmp/sdkdoc/Build/Intermediates.noindex (in target 'Auth0' from project 'Pods')
    cd /redacted-path-to-repo-on-disk/Pods
    builtin-create-build-directory /tmp/sdkdoc/Build/Intermediates.noindex

CreateBuildDirectory /tmp/sdkdoc/Build/Products (in target 'Auth0' from project 'Pods')
    cd /redacted-path-to-repo-on-disk/Pods
    builtin-create-build-directory /tmp/sdkdoc/Build/Products

MkDir /tmp/sdkdoc/Build/Products/Debug-maccatalyst/ReactiveSwift/ReactiveSwift.framework (in target 'ReactiveSwift' from project 'Pods')
    cd /redacted-path-to-repo-on-disk/Pods
    /bin/mkdir -p /tmp/sdkdoc/Build/Products/Debug-maccatalyst/ReactiveSwift/ReactiveSwift.framework

SymLink /tmp/sdkdoc/Build/Products/Debug-maccatalyst/ReactiveSwift/ReactiveSwift.framework/Modules Versions/Current/Modules (in target 'ReactiveSwift' from project 'Pods')
    cd /redacted-path-to-repo-on-disk/Pods
    /bin/ln -sfh Versions/Current/Modules /tmp/sdkdoc/Build/Products/Debug-maccatalyst/ReactiveSwift/ReactiveSwift.framework/Modules

SymLink /tmp/sdkdoc/Build/Products/Debug-maccatalyst/ReactiveSwift/ReactiveSwift.framework/ReactiveSwift Versions/Current/ReactiveSwift (in target 'ReactiveSwift' from project 'Pods')
    cd /redacted-path-to-repo-on-disk/Pods
    /bin/ln -sfh Versions/Current/ReactiveSwift /tmp/sdkdoc/Build/Products/Debug-maccatalyst/ReactiveSwift/ReactiveSwift.framework/ReactiveSwift

SymLink /tmp/sdkdoc/Build/Products/Debug-maccatalyst/ReactiveSwift/ReactiveSwift.framework/Resources Versions/Current/Resources (in target 'ReactiveSwift' from project 'Pods')
    cd /redacted-path-to-repo-on-disk/Pods
    /bin/ln -sfh Versions/Current/Resources /tmp/sdkdoc/Build/Products/Debug-maccatalyst/ReactiveSwift/ReactiveSwift.framework/Resources
```
[snip] And a lot more SymLink and MkDir, omitted for brevity [/snip]
```
WriteAuxiliaryFile /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/all-product-headers.yaml (in target 'SimpleKeychain' from project 'Pods')
    cd /redacted-path-to-repo-on-disk/Pods
    write-file /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/all-product-headers.yaml

WriteAuxiliaryFile /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/SimpleKeychain.hmap (in target 'SimpleKeychain' from project 'Pods')
    cd /redacted-path-to-repo-on-disk/Pods
    write-file /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/SimpleKeychain.hmap
```
[snip] And a lot more WriteAuxiliaryFile on yaml/hmap/modulemap files, omitted for brevity [/snip]
```
ProcessInfoPlistFile /tmp/sdkdoc/Build/Products/Debug-maccatalyst/KeychainAccess/KeychainAccess.framework/Versions/A/Resources/Info.plist /redacted-path-to-repo-on-disk/Pods/Target\ Support\ Files/KeychainAccess/KeychainAccess-Info.plist (in target 'KeychainAccess' from project 'Pods')
    cd /redacted-path-to-repo-on-disk/Pods
    builtin-infoPlistUtility /redacted-path-to-repo-on-disk/Pods/Target\ Support\ Files/KeychainAccess/KeychainAccess-Info.plist -producttype com.apple.product-type.framework -expandbuildsettings -platform macosx -o /tmp/sdkdoc/Build/Products/Debug-maccatalyst/KeychainAccess/KeychainAccess.framework/Versions/A/Resources/Info.plist

CpHeader /redacted-path-to-repo-on-disk/Pods/Target\ Support\ Files/KeychainAccess/KeychainAccess-umbrella.h /tmp/sdkdoc/Build/Products/Debug-maccatalyst/KeychainAccess/KeychainAccess.framework/Versions/A/Headers/KeychainAccess-umbrella.h (in target 'KeychainAccess' from project 'Pods')
    cd /redacted-path-to-repo-on-disk/Pods
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -resolve-src-symlinks /redacted-path-to-repo-on-disk/Pods/Target\ Support\ Files/KeychainAccess/KeychainAccess-umbrella.h /tmp/sdkdoc/Build/Products/Debug-maccatalyst/KeychainAccess/KeychainAccess.framework/Versions/A/Headers
```
[...]
```
Ditto /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/module.modulemap /tmp/sdkdoc/Build/Products/Debug-maccatalyst/KeychainAccess/KeychainAccess.framework/Versions/A/Modules/module.modulemap (in target 'KeychainAccess' from project 'Pods')
    cd /redacted-path-to-repo-on-disk/Pods
    builtin-copy -exclude .DS_Store -exclude CVS -exclude .svn -exclude .git -exclude .hg -resolve-src-symlinks /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/module.modulemap /tmp/sdkdoc/Build/Products/Debug-maccatalyst/KeychainAccess/KeychainAccess.framework/Versions/A/Modules
```
[snip] And some more ProcessInfoPlistFile, CpHeader and Ditto, omitted for brevity [/snip]
</details>

<details>
<summary>Then we get to the CompileC invocations</summary>

```
CompileC /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/Objects-normal/x86_64/SimpleKeychain-dummy.o /redacted-path-to-repo-on-disk/Pods/Target\ Support\ Files/SimpleKeychain/SimpleKeychain-dummy.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler (in target 'SimpleKeychain' from project 'Pods')
    cd /redacted-path-to-repo-on-disk/Pods
    export LANG=en_US.US-ASCII
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -x objective-c -target x86_64-apple-ios13.0-macabi -fmessage-length=229 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit=0 -fcolor-diagnostics -std=gnu11 -fobjc-arc -fobjc-weak -fmodules -gmodules -fmodules-cache-path=/tmp/sdkdoc/ModuleCache.noindex -fmodules-prune-interval=86400 -fmodules-prune-after=345600 -fbuild-session-file=/tmp/sdkdoc/ModuleCache.noindex/Session.modulevalidation -fmodules-validate-once-per-build-session -Wnon-modular-include-in-framework-module -Werror=non-modular-include-in-framework-module -fmodule-name=SimpleKeychain -Wno-trigraphs -fpascal-strings -O0 -fno-common -Wno-missing-field-initializers -Wno-missing-prototypes -Werror=return-type -Wdocumentation -Wunreachable-code -Wno-implicit-atomic-properties -Werror=deprecated-objc-isa-usage -Wno-objc-interface-ivars -Werror=objc-root-class -Wno-arc-repeated-use-of-weak -Wimplicit-retain-self -Wduplicate-method-match -Wno-missing-braces -Wparentheses -Wswitch -Wunused-function -Wno-unused-label -Wno-unused-parameter -Wunused-variable -Wunused-value -Wempty-body -Wuninitialized -Wconditional-uninitialized -Wno-unknown-pragmas -Wno-shadow -Wno-four-char-constants -Wno-conversion -Wconstant-conversion -Wint-conversion -Wbool-conversion -Wenum-conversion -Wno-float-conversion -Wnon-literal-null-conversion -Wobjc-literal-conversion -Wshorten-64-to-32 -Wpointer-sign -Wno-newline-eof -Wno-selector -Wno-strict-selector-match -Wundeclared-selector -Wdeprecated-implementations -DPOD_CONFIGURATION_DEBUG=1 -DDEBUG=1 -DCOCOAPODS=1 -DOBJC_OLD_DISPATCH_PROTOTYPES=0 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -fasm-blocks -fstrict-aliasing -Wprotocol -Wdeprecated-declarations -g -Wno-sign-conversion -Winfinite-recursion -Wcomma -Wblock-capture-autoreleasing -Wstrict-prototypes -Wno-semicolon-before-method-body -Wunguarded-availability -index-store-path /tmp/sdkdoc/Index/DataStore -iquote /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/SimpleKeychain-generated-files.hmap -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/SimpleKeychain-own-target-headers.hmap -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/SimpleKeychain-all-non-framework-target-headers.hmap -ivfsoverlay /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/all-product-headers.yaml -iquote /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/SimpleKeychain-project-headers.hmap -I/tmp/sdkdoc/Build/Products/Debug-maccatalyst/SimpleKeychain/include -isystem /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/iOSSupport/usr/include -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/DerivedSources-normal/x86_64 -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/DerivedSources/x86_64 -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/DerivedSources -F/tmp/sdkdoc/Build/Products/Debug-maccatalyst/SimpleKeychain -iframework /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/iOSSupport/System/Library/Frameworks -include /redacted-path-to-repo-on-disk/Pods/Target\ Support\ Files/SimpleKeychain/SimpleKeychain-prefix.pch -MMD -MT dependencies -MF /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/Objects-normal/x86_64/SimpleKeychain-dummy.d --serialize-diagnostics /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/Objects-normal/x86_64/SimpleKeychain-dummy.dia -c /redacted-path-to-repo-on-disk/Pods/Target\ Support\ Files/SimpleKeychain/SimpleKeychain-dummy.m -o /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/Objects-normal/x86_64/SimpleKeychain-dummy.o

WriteAuxiliaryFile /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/Objects-normal/x86_64/SimpleKeychain.LinkFileList (in target 'SimpleKeychain' from project 'Pods')
    cd /redacted-path-to-repo-on-disk/Pods
    write-file /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/Objects-normal/x86_64/SimpleKeychain.LinkFileList

CompileC /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/Objects-normal/x86_64/A0SimpleKeychain.o /redacted-path-to-repo-on-disk/Pods/SimpleKeychain/SimpleKeychain/A0SimpleKeychain.m normal x86_64 objective-c com.apple.compilers.llvm.clang.1_0.compiler (in target 'SimpleKeychain' from project 'Pods')
    cd /redacted-path-to-repo-on-disk/Pods
    export LANG=en_US.US-ASCII
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -x objective-c -target x86_64-apple-ios13.0-macabi -fmessage-length=229 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit=0 -fcolor-diagnostics -std=gnu11 -fobjc-arc -fobjc-weak -fmodules -gmodules -fmodules-cache-path=/tmp/sdkdoc/ModuleCache.noindex -fmodules-prune-interval=86400 -fmodules-prune-after=345600 -fbuild-session-file=/tmp/sdkdoc/ModuleCache.noindex/Session.modulevalidation -fmodules-validate-once-per-build-session -Wnon-modular-include-in-framework-module -Werror=non-modular-include-in-framework-module -fmodule-name=SimpleKeychain -Wno-trigraphs -fpascal-strings -O0 -fno-common -Wno-missing-field-initializers -Wno-missing-prototypes -Werror=return-type -Wdocumentation -Wunreachable-code -Wno-implicit-atomic-properties -Werror=deprecated-objc-isa-usage -Wno-objc-interface-ivars -Werror=objc-root-class -Wno-arc-repeated-use-of-weak -Wimplicit-retain-self -Wduplicate-method-match -Wno-missing-braces -Wparentheses -Wswitch -Wunused-function -Wno-unused-label -Wno-unused-parameter -Wunused-variable -Wunused-value -Wempty-body -Wuninitialized -Wconditional-uninitialized -Wno-unknown-pragmas -Wno-shadow -Wno-four-char-constants -Wno-conversion -Wconstant-conversion -Wint-conversion -Wbool-conversion -Wenum-conversion -Wno-float-conversion -Wnon-literal-null-conversion -Wobjc-literal-conversion -Wshorten-64-to-32 -Wpointer-sign -Wno-newline-eof -Wno-selector -Wno-strict-selector-match -Wundeclared-selector -Wdeprecated-implementations -DPOD_CONFIGURATION_DEBUG=1 -DDEBUG=1 -DCOCOAPODS=1 -DOBJC_OLD_DISPATCH_PROTOTYPES=0 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -fasm-blocks -fstrict-aliasing -Wprotocol -Wdeprecated-declarations -g -Wno-sign-conversion -Winfinite-recursion -Wcomma -Wblock-capture-autoreleasing -Wstrict-prototypes -Wno-semicolon-before-method-body -Wunguarded-availability -index-store-path /tmp/sdkdoc/Index/DataStore -iquote /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/SimpleKeychain-generated-files.hmap -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/SimpleKeychain-own-target-headers.hmap -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/SimpleKeychain-all-non-framework-target-headers.hmap -ivfsoverlay /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/all-product-headers.yaml -iquote /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/SimpleKeychain-project-headers.hmap -I/tmp/sdkdoc/Build/Products/Debug-maccatalyst/SimpleKeychain/include -isystem /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/iOSSupport/usr/include -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/DerivedSources-normal/x86_64 -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/DerivedSources/x86_64 -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/DerivedSources -F/tmp/sdkdoc/Build/Products/Debug-maccatalyst/SimpleKeychain -iframework /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/iOSSupport/System/Library/Frameworks -w -Xanalyzer -analyzer-disable-all-checks -include /redacted-path-to-repo-on-disk/Pods/Target\ Support\ Files/SimpleKeychain/SimpleKeychain-prefix.pch -MMD -MT dependencies -MF /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/Objects-normal/x86_64/A0SimpleKeychain.d --serialize-diagnostics /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/Objects-normal/x86_64/A0SimpleKeychain.dia -c /redacted-path-to-repo-on-disk/Pods/SimpleKeychain/SimpleKeychain/A0SimpleKeychain.m -o /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/Objects-normal/x86_64/A0SimpleKeychain.o
```
[snip] Some other CompileC invocations that seem to succeed, omitted for brevity [/snip]
</details>

<details>
<summary>Then at some point we encounter some MacCatalyst errors on CompileSwiftSources</summary>

```
CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler (in target 'KeychainAccess' from project 'Pods')
    cd /redacted-path-to-repo-on-disk/Pods
    export DEVELOPER_DIR=/Applications/Xcode.app/Contents/Developer
    export SDKROOT=/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swiftc -incremental -module-name KeychainAccess -Onone -enable-batch-mode -enforce-exclusivity=checked @/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/Objects-normal/x86_64/KeychainAccess.SwiftFileList -DDEBUG -D COCOAPODS -suppress-warnings -sdk /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -target x86_64-apple-ios13.0-macabi -g -module-cache-path /tmp/sdkdoc/ModuleCache.noindex -Xfrontend -serialize-debugging-options -enable-testing -index-store-path /tmp/sdkdoc/Index/DataStore -swift-version 5 -I /tmp/sdkdoc/Build/Products/Debug-maccatalyst/KeychainAccess -F /tmp/sdkdoc/Build/Products/Debug-maccatalyst/KeychainAccess -Fsystem /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/iOSSupport/System/Library/Frameworks -parse-as-library -c -j12 -output-file-map /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/Objects-normal/x86_64/KeychainAccess-OutputFileMap.json -parseable-output -serialize-diagnostics -emit-dependencies -emit-module -emit-module-path /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/Objects-normal/x86_64/KeychainAccess.swiftmodule -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/swift-overrides.hmap -Xcc -iquote -Xcc /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/KeychainAccess-generated-files.hmap -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/KeychainAccess-own-target-headers.hmap -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/KeychainAccess-all-non-framework-target-headers.hmap -Xcc -ivfsoverlay -Xcc /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/all-product-headers.yaml -Xcc -iquote -Xcc /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/KeychainAccess-project-headers.hmap -Xcc -I/tmp/sdkdoc/Build/Products/Debug-maccatalyst/KeychainAccess/include -Xcc -isystem -Xcc /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/iOSSupport/usr/include -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/DerivedSources-normal/x86_64 -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/DerivedSources/x86_64 -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/DerivedSources -Xcc -DPOD_CONFIGURATION_DEBUG=1 -Xcc -DDEBUG=1 -Xcc -DCOCOAPODS=1 -emit-objc-header -emit-objc-header-path /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/Objects-normal/x86_64/KeychainAccess-Swift.h -import-underlying-module -Xcc -ivfsoverlay -Xcc /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/unextended-module-overlay.yaml -working-directory /redacted-path-to-repo-on-disk/Pods
CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/KeychainAccess/Lib/KeychainAccess/Keychain.swift (in target 'KeychainAccess' from project 'Pods')
    cd /redacted-path-to-repo-on-disk/Pods
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift -frontend -c -primary-file /redacted-path-to-repo-on-disk/Pods/KeychainAccess/Lib/KeychainAccess/Keychain.swift -emit-module-path /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/Objects-normal/x86_64/Keychain\~partial.swiftmodule -emit-module-doc-path /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/Objects-normal/x86_64/Keychain\~partial.swiftdoc -serialize-diagnostics-path /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/Objects-normal/x86_64/Keychain.dia -emit-dependencies-path /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/Objects-normal/x86_64/Keychain.d -emit-reference-dependencies-path /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/Objects-normal/x86_64/Keychain.swiftdeps -target x86_64-apple-ios13.0-macabi -enable-objc-interop -sdk /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -I /tmp/sdkdoc/Build/Products/Debug-maccatalyst/KeychainAccess -F /tmp/sdkdoc/Build/Products/Debug-maccatalyst/KeychainAccess -Fsystem /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/iOSSupport/System/Library/Frameworks -enable-testing -g -import-underlying-module -module-cache-path /tmp/sdkdoc/ModuleCache.noindex -suppress-warnings -swift-version 5 -enforce-exclusivity=checked -Onone -D DEBUG -D COCOAPODS -serialize-debugging-options -Xcc -working-directory -Xcc /redacted-path-to-repo-on-disk/Pods -enable-anonymous-context-mangled-names -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/swift-overrides.hmap -Xcc -iquote -Xcc /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/KeychainAccess-generated-files.hmap -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/KeychainAccess-own-target-headers.hmap -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/KeychainAccess-all-non-framework-target-headers.hmap -Xcc -ivfsoverlay -Xcc /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/all-product-headers.yaml -Xcc -iquote -Xcc /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/KeychainAccess-project-headers.hmap -Xcc -I/tmp/sdkdoc/Build/Products/Debug-maccatalyst/KeychainAccess/include -Xcc -isystem -Xcc /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/iOSSupport/usr/include -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/DerivedSources-normal/x86_64 -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/DerivedSources/x86_64 -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/DerivedSources -Xcc -DPOD_CONFIGURATION_DEBUG=1 -Xcc -DDEBUG=1 -Xcc -DCOCOAPODS=1 -Xcc -ivfsoverlay -Xcc /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/unextended-module-overlay.yaml -parse-as-library -module-name KeychainAccess -o /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/KeychainAccess.build/Objects-normal/x86_64/Keychain.o -index-store-path /tmp/sdkdoc/Index/DataStore -index-system-modules
/redacted-path-to-repo-on-disk/Pods/KeychainAccess/Lib/KeychainAccess/Keychain.swift:1175:37: error: 'kSecSharedPassword' is unavailable in Mac Catalyst
private let SharedPassword = String(kSecSharedPassword)
                                    ^~~~~~~~~~~~~~~~~~
Security.kSecSharedPassword:3:12: note: 'kSecSharedPassword' has been explicitly marked unavailable here
public let kSecSharedPassword: CFString
           ^
/redacted-path-to-repo-on-disk/Pods/KeychainAccess/Lib/KeychainAccess/Keychain.swift:886:13: error: 'SecAddSharedWebCredential' is unavailable in Mac Catalyst
            SecAddSharedWebCredential(domain as CFString, account as CFString, password as CFString?) { error -> () in
            ^~~~~~~~~~~~~~~~~~~~~~~~~
Security.SecAddSharedWebCredential:3:13: note: 'SecAddSharedWebCredential' has been explicitly marked unavailable here
public func SecAddSharedWebCredential(_ fqdn: CFString, _ account: CFString, _ password: CFString?, _ completionHandler: @escaping (CFError?) -> Void)
            ^
/redacted-path-to-repo-on-disk/Pods/KeychainAccess/Lib/KeychainAccess/Keychain.swift:931:9: error: 'SecRequestSharedWebCredential' is unavailable in Mac Catalyst
        SecRequestSharedWebCredential(domain as CFString?, account as CFString?) { (credentials, error) -> () in
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Security.SecRequestSharedWebCredential:3:13: note: 'SecRequestSharedWebCredential' has been explicitly marked unavailable here
public func SecRequestSharedWebCredential(_ fqdn: CFString?, _ account: CFString?, _ completionHandler: @escaping (CFArray?, CFError?) -> Void)
            ^
/redacted-path-to-repo-on-disk/Pods/KeychainAccess/Lib/KeychainAccess/Keychain.swift:970:16: error: 'SecCreateSharedWebCredentialPassword()' is unavailable in Mac Catalyst
        return SecCreateSharedWebCredentialPassword()! as String
               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Security.SecCreateSharedWebCredentialPassword:3:13: note: 'SecCreateSharedWebCredentialPassword()' has been explicitly marked unavailable here
public func SecCreateSharedWebCredentialPassword() -> CFString?
            ^

CompileC /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/Objects-normal/x86_64/SimpleKeychain_vers.o /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/DerivedSources/SimpleKeychain_vers.c normal x86_64 c com.apple.compilers.llvm.clang.1_0.compiler (in target 'SimpleKeychain' from project 'Pods')
    cd /redacted-path-to-repo-on-disk/Pods
    export LANG=en_US.US-ASCII
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -x c -target x86_64-apple-ios13.0-macabi -fmessage-length=229 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit=0 -fcolor-diagnostics -std=gnu11 -fmodules -gmodules -fmodules-cache-path=/tmp/sdkdoc/ModuleCache.noindex -fmodules-prune-interval=86400 -fmodules-prune-after=345600 -fbuild-session-file=/tmp/sdkdoc/ModuleCache.noindex/Session.modulevalidation -fmodules-validate-once-per-build-session -Wnon-modular-include-in-framework-module -Werror=non-modular-include-in-framework-module -fmodule-name=SimpleKeychain -Wno-trigraphs -fpascal-strings -O0 -fno-common -Wno-missing-field-initializers -Wno-missing-prototypes -Werror=return-type -Wdocumentation -Wunreachable-code -Werror=deprecated-objc-isa-usage -Werror=objc-root-class -Wno-missing-braces -Wparentheses -Wswitch -Wunused-function -Wno-unused-label -Wno-unused-parameter -Wunused-variable -Wunused-value -Wempty-body -Wuninitialized -Wconditional-uninitialized -Wno-unknown-pragmas -Wno-shadow -Wno-four-char-constants -Wno-conversion -Wconstant-conversion -Wint-conversion -Wbool-conversion -Wenum-conversion -Wno-float-conversion -Wnon-literal-null-conversion -Wobjc-literal-conversion -Wshorten-64-to-32 -Wpointer-sign -Wno-newline-eof -DPOD_CONFIGURATION_DEBUG=1 -DDEBUG=1 -DCOCOAPODS=1 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -fasm-blocks -fstrict-aliasing -Wdeprecated-declarations -g -Wno-sign-conversion -Winfinite-recursion -Wcomma -Wblock-capture-autoreleasing -Wstrict-prototypes -Wno-semicolon-before-method-body -Wunguarded-availability -index-store-path /tmp/sdkdoc/Index/DataStore -iquote /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/SimpleKeychain-generated-files.hmap -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/SimpleKeychain-own-target-headers.hmap -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/SimpleKeychain-all-non-framework-target-headers.hmap -ivfsoverlay /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/all-product-headers.yaml -iquote /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/SimpleKeychain-project-headers.hmap -I/tmp/sdkdoc/Build/Products/Debug-maccatalyst/SimpleKeychain/include -isystem /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/iOSSupport/usr/include -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/DerivedSources-normal/x86_64 -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/DerivedSources/x86_64 -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/DerivedSources -F/tmp/sdkdoc/Build/Products/Debug-maccatalyst/SimpleKeychain -iframework /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/iOSSupport/System/Library/Frameworks -MMD -MT dependencies -MF /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/Objects-normal/x86_64/SimpleKeychain_vers.d --serialize-diagnostics /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/Objects-normal/x86_64/SimpleKeychain_vers.dia -c /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/DerivedSources/SimpleKeychain_vers.c -o /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/SimpleKeychain.build/Objects-normal/x86_64/SimpleKeychain_vers.o
```
[snip] Then some more CompileC invocations without error [/snip]
</details>

<details>
<summary>And finally some nonzero exit code on CompileSwift invocations</summary>

```
CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Disposable.swift (in target 'ReactiveSwift' from project 'Pods')
    cd /redacted-path-to-repo-on-disk/Pods
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift -frontend -c /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Action.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Atomic.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Bag.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Deprecations+Removals.swift -primary-file /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Disposable.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Event.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/EventLogger.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Flatten.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/FoundationExtensions.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Lifetime.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Observer.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Optional.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Property.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Reactive.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/ResultExtensions.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Scheduler.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Signal.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/SignalProducer.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/UnidirectionalBinding.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/UninhabitedTypeGuards.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/ValidatingProperty.swift -emit-module-path /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/Objects-normal/x86_64/Disposable\~partial.swiftmodule -emit-module-doc-path /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/Objects-normal/x86_64/Disposable\~partial.swiftdoc -serialize-diagnostics-path /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/Objects-normal/x86_64/Disposable.dia -emit-dependencies-path /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/Objects-normal/x86_64/Disposable.d -emit-reference-dependencies-path /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/Objects-normal/x86_64/Disposable.swiftdeps -target x86_64-apple-ios13.0-macabi -enable-objc-interop -sdk /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -I /tmp/sdkdoc/Build/Products/Debug-maccatalyst/ReactiveSwift -F /tmp/sdkdoc/Build/Products/Debug-maccatalyst/ReactiveSwift -Fsystem /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/iOSSupport/System/Library/Frameworks -application-extension -enable-testing -g -import-underlying-module -module-cache-path /tmp/sdkdoc/ModuleCache.noindex -suppress-warnings -swift-version 5 -enforce-exclusivity=checked -Onone -D DEBUG -D COCOAPODS -serialize-debugging-options -Xcc -working-directory -Xcc /redacted-path-to-repo-on-disk/Pods -enable-anonymous-context-mangled-names -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/swift-overrides.hmap -Xcc -iquote -Xcc /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/ReactiveSwift-generated-files.hmap -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/ReactiveSwift-own-target-headers.hmap -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/ReactiveSwift-all-non-framework-target-headers.hmap -Xcc -ivfsoverlay -Xcc /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/all-product-headers.yaml -Xcc -iquote -Xcc /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/ReactiveSwift-project-headers.hmap -Xcc -I/tmp/sdkdoc/Build/Products/Debug-maccatalyst/ReactiveSwift/include -Xcc -isystem -Xcc /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/iOSSupport/usr/include -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/DerivedSources-normal/x86_64 -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/DerivedSources/x86_64 -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/DerivedSources -Xcc -DPOD_CONFIGURATION_DEBUG=1 -Xcc -DDEBUG=1 -Xcc -DCOCOAPODS=1 -Xcc -ivfsoverlay -Xcc /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/unextended-module-overlay.yaml -module-name ReactiveSwift -o /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/Objects-normal/x86_64/Disposable.o -index-store-path /tmp/sdkdoc/Index/DataStore -index-system-modules
Command CompileSwift failed with a nonzero exit code

CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Event.swift (in target 'ReactiveSwift' from project 'Pods')
    cd /redacted-path-to-repo-on-disk/Pods
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift -frontend -c /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Action.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Atomic.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Bag.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Deprecations+Removals.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Disposable.swift -primary-file /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Event.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/EventLogger.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Flatten.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/FoundationExtensions.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Lifetime.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Observer.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Optional.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Property.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Reactive.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/ResultExtensions.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Scheduler.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Signal.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/SignalProducer.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/UnidirectionalBinding.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/UninhabitedTypeGuards.swift /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/ValidatingProperty.swift -emit-module-path /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/Objects-normal/x86_64/Event\~partial.swiftmodule -emit-module-doc-path /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/Objects-normal/x86_64/Event\~partial.swiftdoc -serialize-diagnostics-path /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/Objects-normal/x86_64/Event.dia -emit-dependencies-path /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/Objects-normal/x86_64/Event.d -emit-reference-dependencies-path /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/Objects-normal/x86_64/Event.swiftdeps -target x86_64-apple-ios13.0-macabi -enable-objc-interop -sdk /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk -I /tmp/sdkdoc/Build/Products/Debug-maccatalyst/ReactiveSwift -F /tmp/sdkdoc/Build/Products/Debug-maccatalyst/ReactiveSwift -Fsystem /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/iOSSupport/System/Library/Frameworks -application-extension -enable-testing -g -import-underlying-module -module-cache-path /tmp/sdkdoc/ModuleCache.noindex -suppress-warnings -swift-version 5 -enforce-exclusivity=checked -Onone -D DEBUG -D COCOAPODS -serialize-debugging-options -Xcc -working-directory -Xcc /redacted-path-to-repo-on-disk/Pods -enable-anonymous-context-mangled-names -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/swift-overrides.hmap -Xcc -iquote -Xcc /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/ReactiveSwift-generated-files.hmap -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/ReactiveSwift-own-target-headers.hmap -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/ReactiveSwift-all-non-framework-target-headers.hmap -Xcc -ivfsoverlay -Xcc /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/all-product-headers.yaml -Xcc -iquote -Xcc /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/ReactiveSwift-project-headers.hmap -Xcc -I/tmp/sdkdoc/Build/Products/Debug-maccatalyst/ReactiveSwift/include -Xcc -isystem -Xcc /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/iOSSupport/usr/include -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/DerivedSources-normal/x86_64 -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/DerivedSources/x86_64 -Xcc -I/tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/DerivedSources -Xcc -DPOD_CONFIGURATION_DEBUG=1 -Xcc -DDEBUG=1 -Xcc -DCOCOAPODS=1 -Xcc -ivfsoverlay -Xcc /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/unextended-module-overlay.yaml -module-name ReactiveSwift -o /tmp/sdkdoc/Build/Intermediates.noindex/Pods.build/Debug-maccatalyst/ReactiveSwift.build/Objects-normal/x86_64/Event.o -index-store-path /tmp/sdkdoc/Index/DataStore -index-system-modules
Command CompileSwift failed with a nonzero exit code
```
[snip] Some other CompileSwift invocations without nonzero exit codes [/snip]
</details>

<details>
<summary>And finally the "BUILD FAILED" summary</summary>

```
** BUILD FAILED **


The following build commands failed:
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/KeychainAccess/Lib/KeychainAccess/Keychain.swift
    CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Disposable.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Event.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/EventLogger.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Flatten.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/ResultExtensions.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Scheduler.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/Signal.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/SignalProducer.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/ReactiveSwift/Sources/UnidirectionalBinding.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/_ObjectiveAuthenticationAPI.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/_ObjectiveManagementAPI.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/_ObjectiveWebAuth.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/Auth0.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/Auth0Authentication.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/Auth0Error.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/Authentication.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/AuthenticationError.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/AuthProvider.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/AuthSession.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/AuthTransaction.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/BioAuthentication.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/ControllerModalPresenter.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/Credentials.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/CredentialsManager.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/CredentialsManagerError.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/Date.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/Handlers.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/Identity.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/JSONObjectPayload.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/Loggable.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/Logger.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/Management.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/ManagementError.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/NativeAuth.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/NSData+URLSafe.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/NSError+Helper.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/NSURL+Auth0.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/NSURLComponents+OAuth2.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/OAuth2Grant.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/Profile.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/Request.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/Requestable.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/Response.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/ResponseType.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/Result.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/SafariAuthenticationCallback.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/SafariAuthenticationSession.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/SafariSession.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/SafariWebAuth.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/SilentSafariViewController.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/Telemetry.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/TransactionStore.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/UserInfo.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/UserPatchAttributes.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/Users.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/WebAuth.swift
    CompileSwift normal x86_64 /redacted-path-to-repo-on-disk/Pods/Auth0/Auth0/WebAuthError.swift
(59 failures)
```
</details>

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -10,7 +10,9 @@
 
 ##### Bug Fixes
 
-* None.
+* Stop the program immediately if the `xcodebuild` command fails.  
+  [John Fairhurst](https://github.com/johnfairh)
+  [#643](https://github.com/jpsim/SourceKitten/issues/643)
 
 ## 0.29.0
 

--- a/Source/SourceKittenFramework/Module.swift
+++ b/Source/SourceKittenFramework/Module.swift
@@ -48,7 +48,7 @@ public struct Module {
         let yamlPath = URL(fileURLWithPath: path).appendingPathComponent(".build/debug.yaml").path
         guard let yaml = try? Yams.compose(yaml: String(contentsOfFile: yamlPath, encoding: .utf8)),
             let commands = (yaml as Node?)?["commands"]?.mapping?.values else {
-            fputs("SPM build manifest does not exist at `\(yamlPath)` or does not match expected format.", stderr)
+            fputs("SPM build manifest does not exist at `\(yamlPath)` or does not match expected format.\n", stderr)
             return nil
         }
 
@@ -72,7 +72,7 @@ public struct Module {
               let otherArguments = moduleCommand["other-args"]?.array(of: String.self),
               let sources = moduleCommand["sources"]?.array(of: String.self),
               let moduleName = moduleCommand["module-name"]!.string else {
-                fputs("SPM build manifest '\(yamlPath)` does not match expected format.", stderr)
+                fputs("SPM build manifest '\(yamlPath)` does not match expected format.\n", stderr)
                 return nil
         }
         name = moduleName
@@ -129,7 +129,16 @@ public struct Module {
 
         // Executing normal build
         fputs("Running xcodebuild\n", stderr)
-        if let output = XcodeBuild.run(arguments: xcodeBuildArguments, inPath: path),
+        let results = XcodeBuild.launch(arguments: xcodeBuildArguments, inPath: path)
+        if results.terminationStatus != 0 {
+            fputs("Could not successfully run `xcodebuild`.\n", stderr)
+            fputs("Please check the build arguments.\n", stderr)
+            let file = URL(fileURLWithPath: NSTemporaryDirectory()).appendingPathComponent("xcodebuild-\(NSUUID().uuidString).log")
+            _ = try? results.data.write(to: file)
+            fputs("Saved `xcodebuild` log file: \(file.path)\n", stderr)
+            return nil
+        }
+        if let output = results.string,
             let arguments = parseCompilerArguments(xcodebuildOutput: output, language: .swift, moduleName: name),
             let moduleName = moduleName(fromArguments: arguments) {
             self.init(name: moduleName, compilerArguments: arguments)

--- a/Source/SourceKittenFramework/Xcode.swift
+++ b/Source/SourceKittenFramework/Xcode.swift
@@ -37,7 +37,7 @@ internal enum XcodeBuild {
     - returns: `xcodebuild`'s STDERR+STDOUT output combined.
     */
     internal static func run(arguments: [String], inPath path: String) -> String? {
-        return String(data: launch(arguments: arguments, inPath: path, pipingStandardError: true), encoding: .utf8)
+        return launch(arguments: arguments, inPath: path, pipingStandardError: true).string
     }
 
     /**
@@ -47,13 +47,13 @@ internal enum XcodeBuild {
      - parameter path:                Path to run `xcodebuild` from.
      - parameter pipingStandardError: Whether to pipe the standard error output. The default value is `true`.
 
-     - returns: `xcodebuild`'s STDOUT output and, optionally, both STDERR+STDOUT output combined.
+     - returns: `Exec.Result` containing `xcodebuild`'s exit status, STDOUT output and, optionally, both STDERR+STDOUT output combined.
      */
-    internal static func launch(arguments: [String], inPath path: String, pipingStandardError: Bool = true) -> Data {
+    internal static func launch(arguments: [String], inPath path: String, pipingStandardError: Bool = true) -> Exec.Results {
         return Exec.run("/usr/bin/xcodebuild",
                         arguments,
                         currentDirectory: path,
-                        stderr: pipingStandardError ? .merge : .inherit).data
+                        stderr: pipingStandardError ? .merge : .inherit)
     }
 
     /**
@@ -67,8 +67,8 @@ internal enum XcodeBuild {
     internal static func showBuildSettings(arguments xcodeBuildArguments: [String],
                                            inPath: String) -> [XcodeBuildSetting]? {
         let arguments = xcodeBuildArguments + ["-showBuildSettings", "-json"]
-        let outputData = XcodeBuild.launch(arguments: arguments, inPath: inPath, pipingStandardError: false)
-        return try? JSONDecoder().decode([XcodeBuildSetting].self, from: outputData)
+        let results = XcodeBuild.launch(arguments: arguments, inPath: inPath, pipingStandardError: false)
+        return try? JSONDecoder().decode([XcodeBuildSetting].self, from: results.data)
     }
 }
 

